src/status_im/chat/specs.cljs

```diff
@@ -29,4 +29,5 @@
 (s/def :chat/new-request (s/nilable map?))
 (s/def :chat/raw-unviewed-messages (s/nilable vector?))
 (s/def :chat/bot-db (s/nilable map?))
-(s/def :chat/geolocation (s/nilable map?))
\ No newline at end of file
+(s/def :chat/geolocation (s/nilable map?))
+(s/def :chat/message-preview-waiting (s/nilable map?))
```

Comment:
could you add spec for this map, because we will add specs to all other maps later, here we use map? because it was first step of using specs

Reply:
Sure will do